packages/wallets/src/features/cfd/screens/Jurisdiction/jurisdiction-contents/jurisdiction_maltainvest_contents.ts

```diff
@@ -4,7 +4,7 @@ export const getJurisdictionMaltainvestContents = (): TJurisdictionCardItems =>
     contents: {
         financial: [
             {
-                description: 'Synthetics, Forex, Stocks, Stock indices, Commodities, and Cryptocurrencies',
+                description: 'Forex, stocks, stock indices, commodities, cryptocurrencies and synthetic indices.',
```

Comment:
do we need `.` here?
```suggestion
                description: 'Forex, stocks, stock indices, commodities, cryptocurrencies and synthetic indices',
```